pkg/restore-util/client.go

```diff
@@ -99,7 +100,7 @@ func (c *pdClient) SplitRegion(ctx context.Context, regionInfo *RegionInfo, key
 		peer = regionInfo.Leader
 	} else {
 		if len(regionInfo.Region.Peers) == 0 {
-			return nil, errors.New("region does not have peer")
+			return nil, fmt.Errorf("region does not have peer")
```

Comment:
why changing these?

Reply:
What about `NewNoStackError`?

https://godoc.org/github.com/pingcap/errors#NewNoStackError

Reply:
I think this error trace is not necessary.

Reply:
> What about `NewNoStackError`?
> 
> https://godoc.org/github.com/pingcap/errors#NewNoStackError

Seems it does not support error format.

Reply:
Well I find it strange why we need to specifically use the no-stack version, since the `String()` of the error won't include the stack.

Reply:
> Well I find it strange why we need to specifically use the no-stack version, since the `String()` of the error won't include the stack.

yes, I have changed back it to the with-stack version.